contributing.md

```diff
@@ -61,7 +63,7 @@ Before starting major work, you should touch base with the maintainers of Featur
 
      **Ubuntu**:
      ```console
-     sudo apt install openjdk-11-jre openjdk-11-jdk scala graphviz -y
+     sudo apt install openjdk-11-jre openjdk-11-jdk scala pandoc pandoc -y
```

Comment:
pandoc is listed twice - one should be graphviz

Reply:
Fixed